Source/PublicAPI/Deprecated/EThree+Encryption.swift

```diff
@@ -282,7 +369,8 @@ extension EThree {
         return encryptedData
     }
 
-    internal func decryptInternal(data: Data, from publicKey: VirgilPublicKey?) throws -> Data {
+    @available(*, deprecated)
+    internal func oldDecryptInternal(data: Data, from publicKey: VirgilPublicKey?) throws -> Data {
```

Comment:
There is no point marking internal funcs deprecated

Reply:
The point is to silence warnings due to use of deprecated methods inside